examples/community/pipeline_consistency_txt2audio.py

```diff
@@ -0,0 +1,481 @@
+from typing import Any, Dict, Optional, Tuple, Union
+
+import torch
+
+from diffusers.models.embeddings import GaussianFourierProjection, TimestepEmbedding, Timesteps
+from diffusers.models.unets.unet_2d_condition import UNet2DConditionModel, UNet2DConditionOutput
+from diffusers.utils import USE_PEFT_BACKEND, deprecate, scale_lora_layers, unscale_lora_layers
+
+
+class UNet2DConditionGuidedModel(UNet2DConditionModel):
```

Comment:
All of the functionality is the same as UNet2DConditionModel except we have another embedding for guidance which is added to timestep embeddings. I would appreciate feedback on the best way to add support for this if it can be done without creating another class.

If we are okay with another class, it might make sense to get rid of additional code for T2I Adapter, GLIGEN, etc.